packages/sanes-chrome-extension/src/extension/background/actions/createPersona/requestSenderWhitelist.ts

```diff
@@ -0,0 +1,34 @@
+const BLOCKED_URLS_KEY = 'buk';
+const SEPARATOR = ',';
```

Comment:
"," is a valid character in URLs. But I think the blacklisting/whitelisting should happen based on domains anyway, so the separator is good. Postponed: https://github.com/iov-one/ponferrada/issues/187